web/.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+    "semi": true,
```

Comment:
https://prettier.io/docs/en/options.html#semicolons
I'd like to have it at false. It seems semicolons can be omitted and be added by the JS engine before compile.